crates/data-transform/src/main.rs

```diff
@@ -113,12 +114,9 @@ impl ModuleResolver for GrootModuleResolver {
             println!("i am groot, module_name: {}", module_name);
             Ok(self.module_map.get(&module_name).cloned())
         } else {
-            let mapped_address = map_typus_address(address);
-            let module_id = ModuleId::new(mapped_address, id.name().to_owned());
-            if &module_id != id {
-                println!("remapped module from {:#?} to {:#?}", id, module_id);
-            }
-            self.original.get_module(&module_id)
+            Err(IndexerError::DataTransformationError(
```

Comment:
this is just to pass compiler check, I reached out to Bryan on slack and this binary is no longer used.

Reply:
if this binary isn't used lets nuke it

Reply:
agreed and let's do it in a follow-up pr